testsuite/tests/kuadrant/authorino/wristband/conftest.py

```diff
@@ -0,0 +1,110 @@
+    return wristband_secret_name
+
+
+@pytest.fixture(scope="session")
+def certificates(cfssl, wildcard_domain):
+    """Certificate hierarchy used for the wristband tests"""
+    chain = {
+        "signing_ca": CertInfo(ca=True),
+    }
+    return cert_builder(cfssl, chain, wildcard_domain)
+
+
+@pytest.fixture(scope="module")
+def envoy_template():
+    """Envoy with additional route match"""
+    return "envoy_wristband.yaml"
+
+
+@pytest.fixture(scope="module")
+def wristband_endpoint(openshift, authorino, authorization_name):
+    """Authorino oidc wristband endpoint"""
+    return f"http://{authorino.oidc_url}:8083/{openshift.project}/{authorization_name}/wristband"
+
+
+@pytest.fixture(scope="module")
+def authorization(authorization, wristband_secret, wristband_endpoint) -> Authorization:
+    """Add wristband response with the signing key to the AuthConfig"""
+    authorization.responses.add(
+        {
+            "name": "wristband",
+            "wrapper": "envoyDynamicMetadata",
+            "wristband": {
+                "issuer": wristband_endpoint,
+                "tokenDuration": 300,  # default value
+                "signingKeyRefs": [
+                    {
+                        "name": wristband_secret,
+                        "algorithm": "RS256",
+                    }
+                ],
+            },
+        }
+    )
+    return authorization
+
+
+@pytest.fixture(scope="module")
+def wristband_token(client, auth):
+    """Test token acquirement from oidc endpoint"""
+    client.headers = {"Authorization": "Bearer " + auth.token.access_token}
+    response = client.get("/auth")
+    assert response.status_code == 200
+
+    assert response.headers.get("wristband-token") is not None
+    return response.headers["wristband-token"]
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module")
+def authorization2_name(blame, oidc_provider):
+    """Second AuthConfig unique name"""
+    return blame("authz2")
+
+
+@pytest.fixture(scope="module")
+def hostname2(envoy, blame):
+    """Second route for the envoy"""
+    return envoy.add_hostname(blame("envoy2"))
+
+
+@pytest.fixture(scope="module")
+def authorization2(openshift, hostname2, authorization2_name, module_label, wristband_endpoint):
+    """Second AuthConfig with authorino oidc endpoint"""
+    authorization2 = AuthConfig.create_instance(
+        openshift, authorization2_name, None, hostnames=[hostname2], labels={"testRun": module_label}
+    )
+    authorization2.identity.oidc("edge-authenticated", wristband_endpoint)
+    return authorization2
+
+
+@pytest.fixture(scope="module")
+def client2(hostname2, envoy):
+    """Client for second AuthConfig"""
+    client = envoy.client()
+    client.base_url = f"http://{hostname2}"
+    yield client
+    client.close()
+
+
+# pylint: disable=unused-argument
+@pytest.fixture(scope="module", autouse=True)
+def commit(request, commit, certificates, authorization2):
```

Comment:
I am not sure certificates need to be there, they should be requested automatically either way